src/devices/develco.ts

```diff
@@ -673,6 +673,21 @@ const definitions: DefinitionWithExtend[] = [
         exposes: [e.occupancy(), e.battery_low()],
         extend: [develcoModernExtend.addCustomClusterManuSpecificDevelcoGenBasic(), develcoModernExtend.readGenBasicPrimaryVersions()],
     },
+    {
+        whiteLabel: [{vendor: 'Frient', model: 'MOSZB-153', description: 'Motion Sensor 2 Pet', fingerprint: [{modelID: 'MOSZB-153'}]}],
```

Comment:
```suggestion
        whiteLabel: [{vendor: 'Frient', model: 'MOSZB-153', description: 'Motion Sensor 2 Pet']}],
```

This will make the device always detected as Frient, I suggest to remove it.